client/my-sites/site-settings/controller.js

```diff
@@ -117,6 +118,18 @@ export function startSiteOwnerTransfer( context, next ) {
 	next();
 }
 
+export function acceptSiteTransfer( context, next ) {
+	context.primary = (
+		<AcceptSiteTransfer
+			siteId={ context.params.site_id }
+			inviteKey={ context.params.invitation_key }
+			redirectTo={ context.query.nextStep }
```

Comment:
Is `redirectTo` imposed by the backend? Or can we hardcode `/settings/site-transferred/:domain` directly to the component?

Reply:
Aha, gotcha.